hack/dev-setup-local

```diff
@@ -52,13 +53,16 @@ kubectl config use-context gardener-dev \
 # Different base64 implementations have different flags
 MINIKUBE_SEED_KUBECONFIG_B64=$(base64 ${MINIKUBE_SEED_KUBECONFIG} | tr -d '\r\n')
 
+kubectl apply -f example/25-controllerinstallation-local.yaml 
+  kubectl apply -f -
```

Comment:
The `suggestion` feature of github is not so powerful and is not working for multiple lines at the moment.
So, this line should be deleted.


```suggestion
```

Reply:
I will change this anyway due to the previous conversation.